File: API/Shipments/Models/MerchantShipment.cs

namespace Shipments.Models
{
    public class MerchantShipment
    {
        public int Id { get; set; }
        public string  VehicleNumber { get; set; }
        public int Kilometers { get; set; }
        public string? DeliveryStatus { get; set; }
        public string PaymentStatus { get; set; } 
        public DateTime ShipmentDate { get; set ;}
        public double FreightCharges { get; set; }
    }
}

File: UI/EAgroUIApp/src/app/Models/merchant-shipment.ts
export class MerchantShipment {

    constructor(
        public id: number,
        public vehicleNumber: string,
        public kilometers: number,
        public deliveryStatus: string,
        public paymentStatus: string,
        public shipmentDate: string,
        public freightCharges: string
    ) { }
}
